ShiftsLoggerApp.MicVerg/ShiftsLoggerUI/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ShiftsLogger.Models;
+using ShiftsLoggerUI;
+
+var options = new DbContextOptionsBuilder<ShiftContext>()
+            .UseSqlServer("SqlServerConnectionString")
+            .Options;
+
+// Create the ShiftContext (database context)
+ShiftContext shiftContext = new ShiftContext(options);
+
+// Create the ShiftsLoggerService and MenuBuilders with dependencies
+ShiftsLoggerService shiftsLoggerService = new ShiftsLoggerService(shiftContext);
```

Comment:
You don't need to create a context in the UI. I believe this was the reason why my database locked and I had to reboot. After all you're creating instances of the shift context in two different places.